src/software/Localization/voctreeQueryUtility.cpp

```diff
@@ -312,21 +313,21 @@ int main(int argc, char** argv)
 
   }
 
-  for(size_t i = 0; i < allMatches.size(); ++i)
+  for(auto docMatches: allMatches)
   {
-    const auto matches = allMatches[i];
+    const auto matches = docMatches.second;
     bfs::path dirname;
-    POPART_COUT("Camera: " << i);
-    POPART_COUT("query document " << i << " has " << matches.size() << " matches\tBest " << matches[0].id << " with score " << matches[0].score);
+    POPART_COUT("Camera: " << docMatches.first);
+    POPART_COUT("query document " << docMatches.first << " has " << matches.size() << " matches\tBest " << matches[0].id << " with score " << matches[0].score);
     if(withOutput)
     {
       if(not matlabOutput)
```

Comment:
That's my fault, apparently `not, and, or` are not supported on VC, so let's roll back to `! matlabOutput `